EAD/EADSession2/Scripts/validation.js

```diff
@@ -0,0 +1,111 @@
+function validation(){
+	if(!nameValidation()){
+		return false;
+	} else if(!emailValidation(document.signupform)){
+		return false;
+	} else if(!passwordValidation(document.signupform)){
+		return false;
+	} else if(!confirmPasswordValidation()){
+		return false;
+	} else if(!contactValidation()){
+		return false;
+	} else if(document.signupform.Company.value == "None"){
+		alert("Select Company");
+		document.signupform.Company.focus();
+		return false;
+	}
+	return true;
+}
+
+function loginValidation(){
+	if(!emailValidation(document.loginForm)){
+		return false;
+	} else if(!passwordValidation(document.loginForm)){
+		return false;
+	}
+	return true;
+}
+
+function nameValidation(){
+	var myform = document.signupform;
+	var inputAplphabets = /^[A-Za-z]+$/;
+	if(myform.firstName.value.length < 2){
```

Comment:
we are performing same length check twice in single method. Ideally there should be generic methods for validations

Reply:
i.e length , email, phone, password formats